emission/tests/exportTests/TestExportModule.py

```diff
@@ -0,0 +1,120 @@
+from future import standard_library
+standard_library.install_aliases()
+from builtins import *
+import os
+import unittest
+import json
+import bson.json_util as bju
+import pathlib as pl
+import emission.storage.timeseries.abstract_timeseries as esta
+import gzip
+import emission.tests.common as etc
+import emission.pipeline.export_stage as epe
+import emission.storage.pipeline_queries as espq
+import emission.exportdata.export_data as eeed
+import emission.export.export as eee
+
+class TestExportModule(unittest.TestCase):
```

Comment:
you can (and should) have multiple tests per class

Reply:
Apologies for the confusion. I have separated the tests into defined sections within the same class. The newest commit is push and has been tested.